minutes/2019-July.md

```diff
@@ -1,12 +1,16 @@
-## 2019-July Public Board Meeting
+# 2019-July Public Board Meeting
 
 **Present**: Hilmar, Nomi, Heather, Yo, Peter, Chris (remote), Bastian
 
 *Guests*: Malvika Sharan, Daniel Davies (remote), Michael Heuer, Spencer Bliven, Raony Guimaraes, Michael Crusoe, Matúš Kalaš
 
+*Date and Location*: July 25, 12:45pm CEST, Congress Center Basel, Switzerland, in the Dehli Room
```

Comment:
Dehli -> Dehli

Also, I see no harm in adding 2019 to the date here.

Reply:
Thanks, changed both. The typo is on the website, too, BTW.